### ISSUE TITLE
Administrator not receiving requests for feedback from particular user account

### ISSUE THREAD

**Issue by shilowil** [portagenetwork/roadmap#586]
(Will provide more detailed information via Slack!)

An administrator reports that they are unable to receive requests for feedback submitted by a specific user account. The administrator created a test plan and submitted a request for feedback, which the administrator received, but they remain unable to receive requests originating from the user account. 

The user is affiliated with the institution, but the administrator is not receiving emails regarding the request, and the plan does not display in the admin menu under plans.

Tagging @lagoan for triage.

**Comment by aaronskiba** [comment]
I only tested this on UAT. I created a new org, added an email of mine, and enabled the requests for feedback. I also created a new user and assigned the new org to them. I then created a test plan and sent a request for feedback. The feedback request email was successfully sent to the administrator email of the org.

![Screenshot from 2024-01-04 11-41-37](https://github.com/portagenetwork/roadmap/assets/71047780/38a13fec-76e9-4acb-918c-f0e0e2ef8d28)
